fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowTabletStorageFormatCommand.java

```diff
@@ -0,0 +1,135 @@
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.trees.plans.commands;
+
+import org.apache.doris.analysis.StmtType;
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.Env;
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.common.AnalysisException;
+import org.apache.doris.common.ErrorCode;
+import org.apache.doris.common.ErrorReport;
+import org.apache.doris.mysql.privilege.PrivPredicate;
+import org.apache.doris.nereids.trees.plans.PlanType;
+import org.apache.doris.nereids.trees.plans.visitor.PlanVisitor;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.ShowResultSet;
+import org.apache.doris.qe.ShowResultSetMetaData;
+import org.apache.doris.qe.StmtExecutor;
+import org.apache.doris.system.Backend;
+import org.apache.doris.task.AgentClient;
+import org.apache.doris.thrift.TCheckStorageFormatResult;
+
+import com.google.common.collect.Lists;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * admin show tablet storage format command
+ */
+public class ShowTabletStorageFormatCommand extends Command implements NoForward {
+    public static final Logger LOG = LogManager.getLogger(ShowTabletStorageFormatCommand.class);
+    private boolean verbose;
+
+    /**
+     * constructor
+     */
+    public ShowTabletStorageFormatCommand(boolean verbose) {
+        super(PlanType.ADMIN_SHOW_TABLET_STORAGE_FORMAT_COMMAND);
+        this.verbose = verbose;
+    }
+
+    public boolean isVerbose() {
+        return verbose;
+    }
+
+    /**
+     * get Meta for show
+     */
+    public ShowResultSetMetaData getMetaData() {
+        ShowResultSetMetaData.Builder builder = ShowResultSetMetaData.builder();
+        if (verbose) {
+            builder.addColumn(new Column("BackendId", ScalarType.createVarchar(30)))
+                    .addColumn(new Column("TabletId", ScalarType.createVarchar(30)))
+                    .addColumn(new Column("StorageFormat", ScalarType.createVarchar(30)));
+        } else {
+            builder.addColumn(new Column("BackendId", ScalarType.createVarchar(30)))
+                    .addColumn(new Column("V1Count", ScalarType.createVarchar(30)))
+                    .addColumn(new Column("V2Count", ScalarType.createVarchar(30)));
+        }
+        return builder.build();
+    }
+
+    @Override
+    public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        if (!Env.getCurrentEnv().getAccessManager().checkGlobalPriv(ConnectContext.get(), PrivPredicate.ADMIN)
+                && !Env.getCurrentEnv().getAccessManager().checkGlobalPriv(ConnectContext.get(),
```

Comment:
`!Env.getCurrentEnv().getAccessManager().checkGlobalPriv(ConnectContext.get(), PrivPredicate.OPERATOR)` is not in ShowTabletStorageFormatStmt's analyze() method, why need it here?

Reply:
Removed the check. Thanks for pointing it out. Since  this is admin only, the operator should not be allowed.